server/src/internalClusterTest/java/org/opensearch/cluster/routing/allocation/decider/ShardsLimitAllocationDeciderRemoteStoreEnabledIT.java

```diff
@@ -313,9 +316,27 @@ public void testCombinedIndexAndClusterPrimaryShardLimits() throws Exception {
                 assertTrue("No node should have more than 3 primary shards total", count <= 3);
             }
         });
+        cleanUp(new String[] { "test1", "test2" });
     }
 
     private void updateClusterSetting(String setting, int value) {
         client().admin().cluster().prepareUpdateSettings().setTransientSettings(Settings.builder().put(setting, value)).get();
     }
+
+    private void cleanUp(String[] indices) throws Exception {
```

Comment:
Use varargs here to make the calling syntax more concise:

```suggestion
    private void cleanUp(String... indices) throws Exception {
```

Reply:
acknowledged and updated